### DISCUSSION TITLE
Way to log Raw DynamoDB Query

### DISCUSSION THREAD

**Discussion by romeosunio** [sensedeep/dynamodb-onetable#425] [Q&A] [answered]
Is there a way to log / printout raw dynamodb queries? Just wanted to know what is happening under the hood when performing `find` with filters and stuff.

**Comment by mobsense** [comment, marked as answer]
Add {log: true} to the API params.

and setup a logger when constructing the Table. See **logger** on https://doc.onetable.io/api/table/constructor/constructor/

**Reply by romeosunio** [reply]
You mean `logger` right?

I have set that to logger to true. But not logging the dynamo queries. Is there a function i need to call/use to generate that raw dynamodb queries?

**Reply by mobsense** [reply]
When logging is enabled, you see the RAW dynamodb query parameters in the log trace.
